### PR TITLE
Delete

### LINKED ISSUE #4: [2pts] Delete Item link in navigation
Create a link to delete item to navigation

### PR DESCRIPTION
**Pull request by brhanuhailu**
- [ ] add delete item
- [ ] Linked to navbar
image

### CODE REVIEW

File: src/components/Delete.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+  useEffect(() => {
+  dispatch(userHouses());
+  },[dispatch]);
+
+  const userhouses = useSelector((state) => state.Houses.userhouses);
+  const navigate = useNavigate();
+
+  const handleDelete = (houseId) => {
+    dispatch(deleteHouseThunk(houseId)).then((action) => {
```
Comment (yashodhicy) [inline src/components/Delete.jsx]: add this suggetions to update the user houses list.
```suggestion
await dispatch(deleteHouseThunk(houseId)).then((action) => {
    dispatch(userHouses())
```

File: src/components/Delete.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  };
+
+  return (
+    <div>
+      <section className="houses">
+      <div className="header">
+        <h2>Need a House to Delete ?</h2>
+        <p>checkout our available Houses</p>
+      </div>
+
```
Comment (yashodhicy) [inline src/components/Delete.jsx]:
```suggestion
      {userhouses.length === 0 ? (
      <p>You have no houses yet. Add a new house.</p>
      ) : (
```

File: src/components/Delete.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            </ul>
+
+            <div>
+              <button type = "button" className="btn" onClick={() => handleDelete(e.id)}>Delete House</button>
+            </div>
+
+          </div>
+
+        ))}
+      </div>
```
Comment (yashodhicy) [inline src/components/Delete.jsx]:
```suggestion
      </div>)}
```

File: src/Redux/HouseSlice.js [hunk 4]
```diff
@@ -30,15 +31,40 @@ export const Deletehouse = createAsyncThunk('house/Deletehouse', async (payload)
...
-});
+})
 
 const initialState = {
   houses: [],
   house: '',
-  userHouses: [],
+  userhouses: [{
+    id: 1,
+    user_id: 2,
```
Comment (yashodhicy) [inline src/Redux/HouseSlice.js]: it is better to remove mock data since our user houses API part is working.

**Review by yashodhicy — CHANGES_REQUESTED** [review]
# Request changes

you have done a great job here but it is better to do these changes I mentioned in the comments.

File: src/components/Delete.jsx (hunk 1 above)
Comment (brhanuhailu) [reply src/components/Delete.jsx]: It is done. can I have little description sis

Comment (yashodhicy) [reply src/components/Delete.jsx]: yeah sure. This is for after deletion it needs to get the newest list of user houses or else it will give an error.

**Review by Mahamadou-Nouridine — APPROVED** [review]
Your changes are approved, you  can merge the Pull request
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c31756761384f6250426e49426f5855512f67697068792e676966

**Pull request merged**